Add initial README.md (#1)
open-telemetry/sig-contributor-experience: README.md

# <img src="https://opentelemetry.io/img/logos/opentelemetry-logo-nav.png" alt="OpenTelemetry Icon" width="45" height=""> OpenTelemetry Contributor Experience SIG

## What is the Contributor Experience SIG?

> The goal of this SIG is to improve the experience of contributors to the OpenTelemetry project. 

Reviewer: Project or projects, as in all SIG code projects on GH?

Author: ```suggestion
The goal of this SIG is to improve the experience of contributors to the OpenTelemetry Project. 
```

Author: otel as a project itself, does making the p uppercase clarify that?

> The SIG creates programs and processes and consults other SIGs to improve the overall community health. 

Reviewer 2: ```suggestion
To achieve this, we create programs and processes around key contribution stages, such as first commit, or the transition from regular maintainer to approver.  We also consult with other SIGs to feedback into the process with an aim to improve overall community health.
```

> To achieve this, we create programs and processes around key contribution stages, such as first commit, or the transition from regular maintainer to approver.  We also consult with other SIGs to feedback into the process with an aim to improve overall community health.

Reviewer 3: This sounds a bit odd (is the intention to highligiht a "downgrade" of role from maintainer to approver?). 
Maybe it should be:
```suggestion
To achieve this, we create programs and processes around key contribution stages, such as first commit, or the transition from regular contributor to approver.  We also consult with other SIGs to feedback into the process with an aim to improve overall community health.
```

Author: ah, lol... brain not good! Thanks

> ## Get Involved

Reviewer: ```suggestion
## Get involved
```

> There are many ways to get involved in our work:

Reviewer: ```suggestion
There are many ways to get involved:
```

> - Create [issues]([github.com/open-telemetry/sig-contributor-experience/issues/](https://github.com/open-telemetry/sig-contributor-experience/issues/new)), comment on [existing issues](github.com/open-telemetry/sig-contributor-experience/issues/), [review pull requests](https://github.com/open-telemetry/sig-contributor-experience/pulls)

Reviewer 4: ```suggestion
- Create [issues]([github.com/open-telemetry/sig-contributor-experience/issues/](https://github.com/open-telemetry/sig-contributor-experience/issues/new)), comment on [existing issues](github.com/open-telemetry/sig-contributor-experience/issues/), or [review pull requests](https://github.com/open-telemetry/sig-contributor-experience/pulls).
```

> - Create [issues]([github.com/open-telemetry/sig-contributor-experience/issues/](https://github.com/open-telemetry/sig-contributor-experience/issues/new)), comment on [existing issues](github.com/open-telemetry/sig-contributor-experience/issues/), or [review pull requests](https://github.com/open-telemetry/sig-contributor-experience/pulls).

Reviewer 5: ```suggestion
- Create [issues](https://github.com/open-telemetry/sig-contributor-experience/issues/new), comment on [existing issues](https://github.com/open-telemetry/sig-contributor-experience/issues/), or [review pull requests](https://github.com/open-telemetry/sig-contributor-experience/pulls).
```

> - Get in touch with us on the [CNCF slack](https://slack.cncf.io/) channel [#otel-contributor-experience](https://cloud-native.slack.com/archives/C06TMJ2R0SK)

Reviewer 4: ```suggestion
- Get in touch with us on the [CNCF slack](https://slack.cncf.io/) channel [#otel-contributor-experience](https://cloud-native.slack.com/archives/C06TMJ2R0SK).
```

> - Create [issues]([github.com/open-telemetry/sig-contributor-experience/issues/](https://github.com/open-telemetry/sig-contributor-experience/issues/new)), comment on [existing issues](github.com/open-telemetry/sig-contributor-experience/issues/), or [review pull requests](https://github.com/open-telemetry/sig-contributor-experience/pulls).

Reviewer: Consider splitting into three bullet list items.

Author: I wanted to avoid having a sub list of items, it looks convoluted in the markdown, because of the links, but the rich view is ok

> - [Attend on of our meetings](https://docs.google.com/document/d/1CTQI0p3QF8JP8reV8z_ggcs8KE5YVPpQGvAQknw4qP0/edit#heading=h.x3xl5syz15zx), we alternate between two times to make it possible for you to join no matter which time zone you live or work in! 

Reviewer 6: ```suggestion
- [Attend one of our meetings](https://docs.google.com/document/d/1CTQI0p3QF8JP8reV8z_ggcs8KE5YVPpQGvAQknw4qP0/edit#heading=h.x3xl5syz15zx), we alternate between two times to make it possible for you to join no matter which time zone you live or work in! 
```

> - [Attend one of our meetings](https://docs.google.com/document/d/1CTQI0p3QF8JP8reV8z_ggcs8KE5YVPpQGvAQknw4qP0/edit#heading=h.x3xl5syz15zx), we alternate between two times to make it possible for you to join no matter which time zone you live or work in! 

Reviewer 4: ```suggestion
- [Attend one of our meetings](https://docs.google.com/document/d/1CTQI0p3QF8JP8reV8z_ggcs8KE5YVPpQGvAQknw4qP0/edit#heading=h.x3xl5syz15zx). We alternate between two times to make it possible for you to join no matter which time zone you live or work in! 
```

> [!NOTE]

> > This SIG operates **async** by default, attending any meetings is not mandatory! Feel free to choose your preferred way of communication

Reviewer 5: ```suggestion
> This SIG operates **async** by default, attending meetings is not mandatory! Feel free to choose your preferred way of communication.
```

Author: ```suggestion
> This SIG operates **async** by default, so attending meetings is not mandatory! Feel free to choose your preferred way of communication.
```

Author: merging @tiffany76's and @kaylareopelle's suggestions

> > This SIG operates **async** by default, attending any meetings is not mandatory! Feel free to choose your preferred way of communication

Reviewer 4: ```suggestion
> This SIG operates **async** by default, so attending any meetings is not mandatory! Feel free to choose your preferred way of communication
```
